packages/phoenix-event-display/src/tests/event-display.spec.ts

```diff
@@ -1,359 +0,0 @@
-import { EventDisplay } from '../event-display';
```

Comment:
There are no tests for the event display?

Reply:
I hope to add some along with the `src/managers` one.